### ISSUE TITLE
Saving a deck for TTS importation - generate error of code - missing arguments

### ISSUE THREAD

**Issue by nsevigny77** [fafranco82/swdestinydb#160]
Hi I didnt see any indications about this problem - with any decks produced with EMpire at war or 2 players starter set, when saving it is seems good in the tts format ; but on the other side, when trying to open the saved objects, it generate that error...

We are on the eve to participate in the Artificery.com event registration end tomorrow, and we cant be assured that this essential functionality will be there ...

You are doing great work, if there is way to donate or contribute, I couldnt live anymore without this integration - keep on good work !

**Comment by fafranco82** [comment]
Yes, I don't have yet the TTS id for cards from EaW or even Two-Player Game... I don't play on TTS and the ids were added by other user... I don't know if he is still active on github and in destiny...

**Comment by fafranco82** [comment]
I've just found out how to get the TTS card ids... try now to export deck to TTS and tell me. I'm not sure if it is right.

**Comment by nsevigny77** [comment]
HI cant answer before coming back to home, I did a quick test, and everyhthing seem to load as almost normal, as it was before new expansion set... Beware that you must too, confirm onSWDestinydb.com that you want to load the new version of cards list...

I say almost because before that we often have to retrieve a dice that was like unknown to TTS ; at this time, it invoked the Custom dice panel on TTS part, and dice disapear after that ; we just have in that case   to retrieve as I said the faulty dice in the dice bag....

Before we were having problem with Baby vader dice, Unkar, Luke lightsaber, and now the dice lost was Mace Mindu... It is rather annoying but knowing that, you put the dice in the saved object, re-save the object and you are fine for other times...

Thanks you again - guys at Artificery.com season league will be very happy to know that TTS is back on its feet !
